src/platform/forms/preSubmitInfo.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  error: 'You must accept the privacy policy before continuing.'
```

Comment:
This should probably be done across applications, but I'm also wondering if we should just see if USFS will change it in their code (we just added a period).

Reply:
The latter sounds good, would it be helpful to set up a ticket on the USFS repo?